tsl/src/fdw/relinfo.c

```diff
@@ -203,6 +201,9 @@ estimate_chunk_fillfactor(Chunk *chunk, Hyperspace *space)
 		/* if we are beyond end range then chunk can possibly be totally filled */
 		if (time_slice->fd.range_end <= now_internal_time)
 		{
+			int total_slices = get_total_number_of_slices(space);
+			int num_created_after = ts_chunk_num_of_chunks_created_after(chunk, total_slices);
```

Comment:
Maybe a simpler way of doing this is just to get the max(chunk_id) and compare it to the chunk_id of the current chunk, assuming that it is always increasing.

num_created_after = max_chunk_id - current_chunk_id;

Reply:
There's no way to get the max chunk id for a given hypertable, unfortunately.